subgraph.yaml

```diff
@@ -46,6 +46,29 @@ dataSources:
           handler: handleDeploySpigot
         - event: DeployedEscrow(indexed address,indexed uint32,indexed address,address)
           handler: handleDeployEscrow
+  - name: FeedRegistry
+    kind: ethereum/contract
+    network: goerli
+    source:
+      abi: FeedRegistry
+      address: "0x4DF0abBe72A79cC65aDf694DE64535904aB43aAc"
+      startBlock: 7966450
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      file: ./src/feed-registry.ts
+      entities:
+        - CollateralToken
```

Comment:
Token
SupportedToken
Oracle

Reply:
Great catch - I had this in my Chainlink Feed Registry subgraph on mainnet and forgot to port this small change over.